### PR TITLE
Actuatlize info about modules in getting-started

### PR DESCRIPTION
**Pull request by myshov**
At this moment the support of esm in browsers is pretty decent — [87%](https://caniuse.com/#search=modules).

### CODE REVIEW

File: src/content/guides/getting-started.md [hunk 1]
```diff
@@ -207,7 +208,7 @@ W> If you are getting a syntax error in the middle of minified JavaScript when o
 
 ## Modules
 
-The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). Although they are not supported in most browsers yet, webpack does support them out of the box.
+The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). They are suppported in the most browsers at this moment, however there are some browsers that don't recognize the new syntax. But don't worry, webpack does support them out of the box.
```
Comment (montogeek) [inline src/content/guides/getting-started.md]:
```suggestion
The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/docs/en/learn/). They are suppported in most of the browsers at this moment, however there are some browsers that don't recognize the new syntax. But don't worry, webpack does support them out of the box.
```

**Comment by myshov** [comment]
fixed

**Comment by myshov** [comment]
@montogeek ping

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**